lib/controller/AuthController.js

```diff
@@ -104,6 +108,9 @@ AuthController.push({
   path: '/auth',
   options: {
     auth: { mode: 'try' },
+    response: {
```

Comment:
can we at least test this in the frontend quickly?  might as well, since it's easier to correct at this stage